### PR TITLE
Design Browser History

### PR DESCRIPTION
**Pull request by sumanth-botlagunta**
1472. Design Browser History solved in python and java

### CODE REVIEW

**Review by sumanth-botlagunta — COMMENTED** [review]
this solution for 1472. Design Browser History is added in java and python

File: python/design_browser_history.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        self.history = self.history[:self.ptr]
+        self.history.append(url)
+               
+    def back(self, steps: int) -> str:
+        self.ptr = max(0, self.ptr - steps)
+        return self.history[self.ptr]
+
+    def forward(self, steps: int) -> str:
+        self.ptr = min(len(self.history) - 1, self.ptr + steps)
+        return self.history[self.ptr]
```
Comment (anishLearnsToCode) [inline python/design_browser_history.py:18]: __[no newline at end of file]__ Can you please add a newline character at the end of the file `\n`.

File: src/DesignBrowserHistory.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    public String back(int steps) {
+        this.ptr = Math.max(0, this.ptr - steps);
+        return this.history.get(this.ptr);
+    }
+
+    public String forward(int steps) {
+        this.ptr = Math.min(this.history.size() - 1, this.ptr + steps);
+        return this.history.get(this.ptr);
+    }
+}
```
Comment (anishLearnsToCode) [inline src/DesignBrowserHistory.java:26]: __[no newline at end of file `\n`]__ can you please add newline character at the end of the file

**Comment by anishLearnsToCode** [comment]
Thank you very much for your contribution, as soon as you have resolved the 2 small comments that I have left on this pr, I will merge it 👍

File: python/design_browser_history.py (hunk 1 above)
Comment (sumanth-botlagunta) [reply python/design_browser_history.py]: [Solved] added new line at end of file

File: src/DesignBrowserHistory.java (hunk 2 above)
Comment (sumanth-botlagunta) [reply src/DesignBrowserHistory.java]: [Solved] added new line at end of file

**Comment by sumanth-botlagunta** [comment]
Solved problem 211 and added new line to all the files

**Pull request merged**